### ISSUE TITLE
Course details page

### ISSUE THREAD

**Issue by anusharish** [codeshaala/codeshaala-dev#14]
Tabular columns with the course batches should be added in place of "key points" section.
It should have below details 
- Batch name
- Mode of training - online or inclass
- Schedule

**Comment by anusharish** [comment]
There should be a table with all the details.

**Comment by rajkamal89** [comment]
Created a table for courses batches.
Have a look.
https://codeshaala.github.io/codeshaala-dev/android-development.html

If any changes required . let me know.
Then for all the course pages i'll create.

**Comment by anusharish** [comment]
This looks good. @sunil3590 what do you think?

**Comment by sunil3590** [comment]
@anusharish @rajkamal89 yes this looks good. we may want to center align the column headers and make them bold/slightly larger/different color. Just to distinguish between header row and other rows

**Comment by rajkamal89** [comment]
@sunil3590 As of now I have only changed the Background color of the Headings.
Can do heading in Center too But Content below heading will not look good with left align.
